wsmaster/codenvy-hosted-system-ram/src/main/java/com/codenvy/service/systemRam/SystemRamLimitChecker.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.service.systemRam;
+
+/**
+ * Checks system RAM limit status.
```

Comment:
This seems to be wrong, as the interface has the only one method which has nothing to do with the ram check at all.